docs/reference/commandline/context_inspect.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "context inspect"
+description: "The context inspect command description and usage"
+keywords: "context, inspect"
+---
+
+<!-- This file is maintained within the docker/cli GitHub
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# context inspect
+
+```markdown
+Usage:  docker context inspect [OPTIONS] [CONTEXT] [CONTEXT...]
+
+Display detailed information on one or more contexts
+
+Options:
+  -f, --format string   Format the output using the given Go template
+```
+
+## Description
+
+Inspects one or more context. Example output:
```

Comment:
typo;

```suggestion
Inspects one or more contexts. Example output:
```

or

```suggestion
Inspects the specified contexts.

By default, this renders all results in a JSON array. If a format is specified,
the given template will be executed for each result.

Go's [text/template](http://golang.org/pkg/text/template/) package
describes all the details of the format.

Example output:
```

I think the "Example output" would more suitable for a short  `Examples` heading https://github.com/docker/cli/blob/master/docs/reference/commandline/service_inspect.md#examples

Could be rewritten to a short example, similar to https://github.com/docker/cli/blob/master/docs/reference/commandline/service_inspect.md#inspect-a-service-by-name-or-id